src/custom-elements/src/components/badge/badge.tsx

```diff
@@ -31,14 +31,14 @@ export class Badge {
   @Prop({ reflectToAttr: true }) textTransform: string;
 
   /**
-   *  to set size of a badge { small or smaller }.
+   *  to set size of a badge { xs or sm }.
    */
   @Prop({ reflectToAttr: true }) size: string;
 
   @Watch('size')
   sizeValidation(newValue: string) {
-    if (newValue && !['', 'small', 'smaller'].includes(newValue)) {
-      throw new Error(`${newValue} is not a valid size for badge. Valid values are small, smaller or ''. `);
+    if (newValue && !['', 'small', 'smaller', , 'sm', 'xs'].includes(newValue)) {
```

Comment:
```suggestion
    if (newValue && !['', 'small', 'smaller', 'sm', 'xs'].includes(newValue)) {
```
Please remove the unnecessary comma